astropy/units/tests/test_equivalencies.py

```diff
@@ -234,6 +291,14 @@ def test_is_equivalent():
     assert not u.L.is_equivalent((u.m, u.s, u.kg))
     assert not (u.km / u.s).is_equivalent((u.m, u.s, u.kg))
 
+    # geometrized equivalencies
+    assert u.kg.is_equivalent(
+        (u.m, u.s), equivalencies=u.geometrized('mass'))
```

Comment:
These tests are best split in 2, by `pytest.mark.parametrize`ing the function for `u.m` and `u.s` - as is, it will just check `u.m`, see that it is equivalent, and return `True`